src/App.tsx

```diff
@@ -15,71 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedItem: string;
```

Comment:
Will be awesome to create enum

Reply:
I'm not quite sure, how exactly should I use enum in this task:/

Reply:
Optional: If you have limited count of products, you could create `enum Goods` and use it instead of `string` type.